components/LandingPageExpo.js

```diff
@@ -0,0 +1,108 @@
+import React, { Component } from 'react';
+import ReactNative, {Button, NativeEventEmitter, Platform} from 'react-native';
+import client, { Color } from '@doubledutch/rn-client'
+const { TouchableOpacity, TouchableHighlight, Text, View, Image, WebView, Dimensions, Linking } = ReactNative
+
+export default class LandingPageExpo extends Component {
+  constructor(props) {
+    super()
+    this.state = {
+      userColor: client.primaryColor 
+    }
+  }
+
+  render() {
+    let customColor = this.state.userColor
+    if (this.props.color !== undefined) {
+      customColor = this.props.color
+    }
+    const colorStyle = {
+      color: customColor
+    }
+    const { video, headline, title, des } = this.props
+    const dimensionStyle = {
+    height : Dimensions.get('window').width * .5625,
```

Comment:
Let's prefer flex layout

Reply:
What would be a good way to maintain the height width ratio via flex?